src/boot/efi/addon.c

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "efi.h"
+
+/* This is intended to carry data, not to be executed */
+
+EFIAPI EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *system_table);
+EFIAPI EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *system_table) {
+    return EFI_SECURITY_VIOLATION;
```

Comment:
shouldn't this output a recognizable error message? Like that message that running old win16 binaries gives you when you invoke it from dos?

Reply:
Yes, I am very old.

Reply:
Lul. That is would be the job of the DOS header. And we don't put one in our PE binaries because nobody gives a damn about those.

Reply:
oh, come on, admit it, it would be awesome if we had a single binary you can run in DOS mode and it tells you to go fuck yourself that it is an EFI binary and then you run it in EFI mode you get a message to go fuck yourself that it's actually an addon.

Reply:
You write the DOS stub then. But no cheating by copy pastaing any of the existing DOS stub code on teh internets. 😛

(Seriously, though, It saves us some bytes: we get more space for more section headers while keeping the header section on one page. Not that it really matters...)